util/phil.py

```diff
@@ -115,7 +115,10 @@ def from_string(self, s):
     if s not in self.cache:
       if not exists(s):
         raise Sorry('File %s does not exist' % s)
-      self.cache[s] = FilenameDataWrapper(s, flex.reflection_table.from_pickle(s))
+      try:
+        self.cache[s] = FilenameDataWrapper(s, flex.reflection_table.from_msgpack_file(s))
+      except:
```

Comment:
Catch a more specific exception here, at a bare minimum make it ```except Exception```.
The difference between this and ```except Exception``` is that this does catch Ctrl+C and ```except Exception``` does not.

Reply:
I have now fixed this